linkerd/app/outbound/src/tcp/logical.rs

```diff
@@ -122,39 +175,74 @@ impl<C> Outbound<C> {
     }
 }
 
-// === impl ProfileRoute ===
+// === impl Params ===
+
+impl From<(Profile, Logical)> for Params {
```

Comment:
nit: _maaaaaybe_ worth a comment saying where this gets impl gets consumed? not a huge deal though.